src/main/java/com/ait/lienzo/client/core/shape/wires/handlers/WiresConnectorControl.java

```diff
@@ -16,7 +18,28 @@
 
     void move(double dx, double dy, boolean midPointsOnly, boolean moveLinePoints);
 
-    void addControlPoint( double x, double y );
+    /**
+     * Add a control point on the existing {@link WiresConnector#getLine()} returning the selected index to the control point
+     * @param x position
+     * @param y position
+     * @return index to the added Control Point
+     */
+    int addControlPoint( double x, double y );
```

Comment:
I would delete spaces in new code before and after formal parameters as it is in other methods.